src/microns_to_nwb/convert_microns_to_nwb.py

```diff
@@ -0,0 +1,36 @@
+from warnings import warn
+
+from phase3 import nda
+
+from tools.behavior import add_eye_tracking, add_treadmill, find_earliest_timestamp
+from tools.intervals import add_trials
+from tools.nwb_helpers import start_nwb
+from tools.ophys import add_ophys
+from tools.stimulus import add_stimulus
+
+
+def build_nwb(scan_key):
+    nwb = start_nwb(scan_key)
+    add_stimulus(scan_key, nwb)
```

Comment:
Looks like the OpticalSeries also has timestamps - do these need to be adjusted?

Reply:
Thanks Cody! I fixed it.